modules/apps/asset/asset-categories-item-selector-web/src/main/java/com/liferay/asset/categories/item/selector/web/internal/display/context/SelectAssetCategoryInfoItemDisplayContext.java

```diff
@@ -70,7 +72,19 @@ public SelectAssetCategoryInfoItemDisplayContext(
 
 	public Map<String, Object> getData() throws Exception {
 		return HashMapBuilder.<String, Object>put(
-			"addCategoryURL", _getAddCategoryURL()
+			"addCategoryURL",
+			() -> {
+				try {
+					return _getAddCategoryURL();
+				}
+				catch (Exception exception) {
+					if (_log.isDebugEnabled()) {
+						_log.debug(exception);
```

Comment:
@AliciaGarciaGarcia, it was this 😞. Now a _log.debug(exception, exception) is transformed to _log.debug(exception)